src/modules/Dropdown/Dropdown.js

```diff
@@ -506,45 +598,44 @@ export default class Dropdown extends Component {
   // ----------------------------------------
 
   renderText = () => {
-    const { options, search, text } = this.props
+    const { multiple, placeholder, search, text } = this.props
     const { searchQuery, value } = this.state
+    const hasValue = multiple ? !_.isEmpty(value) : !!value
 
-    const classes = cx('text', search && searchQuery && 'filtered')
+    if (multiple) return
 
-    let _text
-    if (text) {
+    const classes = cx(
+      placeholder && !hasValue && 'default',
+      'text',
+      search && searchQuery && 'filtered'
+    )
+    let _text = placeholder
```

Comment:
`text` is already destructured in this context.  The final rendered value can be `this.props.text`, `null`, or the currently selected item's `text` prop (see logic below).

Suppose it could be something like `renderedText`?